diff --git a/src/evhttp/http.c b/src/evhttp/http.c
index 6774a32..09ac677 100644
--- a/src/evhttp/http.c
+++ b/src/evhttp/http.c
@@ -25,8 +25,6 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <event-config.h>
-
 #ifdef _EVENT_HAVE_SYS_PARAM_H
 #include <sys/param.h>
 #endif
@@ -71,17 +69,17 @@
 #endif
 #include <signal.h>
 #include <time.h>
+
+#undef timeout_pending
+#undef timeout_initialized
+
+#include <event.h>
 #ifdef _EVENT_HAVE_UNISTD_H
 #include <unistd.h>
 #endif
 #ifdef _EVENT_HAVE_FCNTL_H
 #include <fcntl.h>
 #endif
-
-#undef timeout_pending
-#undef timeout_initialized
-
-#include <event.h>
 #include "evhttp.h"
 #include <evutil.h>
 #include "log.h"
diff --git a/src/evrtsp/rtsp.c b/src/evrtsp/rtsp.c
index 242dbe4..dc6ef10 100644
--- a/src/evrtsp/rtsp.c
+++ b/src/evrtsp/rtsp.c
@@ -28,21 +28,6 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <event-config.h>
-
-#ifdef _EVENT_HAVE_SYS_PARAM_H
-#include <sys/param.h>
-#endif
-#ifdef _EVENT_HAVE_SYS_TYPES_H
-#include <sys/types.h>
-#endif
-
-#ifdef _EVENT_HAVE_SYS_TIME_H
-#include <sys/time.h>
-#endif
-#ifdef _EVENT_HAVE_SYS_IOCCOM_H
-#include <sys/ioccom.h>
-#endif
 
 #ifndef WIN32
 #include <sys/resource.h>
@@ -74,6 +59,26 @@
 #endif
 #include <signal.h>
 #include <time.h>
+
+#undef timeout_pending
+#undef timeout_initialized
+
+#include <event.h>
+
+#ifdef _EVENT_HAVE_SYS_PARAM_H
+#include <sys/param.h>
+#endif
+#ifdef _EVENT_HAVE_SYS_TYPES_H
+#include <sys/types.h>
+#endif
+
+#ifdef _EVENT_HAVE_SYS_TIME_H
+#include <sys/time.h>
+#endif
+#ifdef _EVENT_HAVE_SYS_IOCCOM_H
+#include <sys/ioccom.h>
+#endif
+
 #ifdef _EVENT_HAVE_UNISTD_H
 #include <unistd.h>
 #endif
@@ -81,10 +86,6 @@
 #include <fcntl.h>
 #endif
 
-#undef timeout_pending
-#undef timeout_initialized
-
-#include <event.h>
 #include "evrtsp.h"
 #include <evutil.h>
 /* #define USE_DEBUG */
